File: app/src/main/java/ru/alexsuvorov/paistewiki/model/SupportAnatomy.java
package ru.alexsuvorov.paistewiki.model;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.Ignore;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

import lombok.Getter;
import lombok.Setter;

@Entity(tableName = "support_anatomy")
public class SupportAnatomy {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "anatomy_id")
    @Getter @Setter @NonNull
    public long anatomyId = 1;

    @ColumnInfo(name = "anatomy_title")
    @Getter @Setter
    public String anatomyTitle;

    @ColumnInfo(name = "anatomy_subtitle")
    @Getter @Setter
    public String anatomySubtitle;

    @ColumnInfo(name = "anatomy_text")
    @Getter @Setter
    public String anatomyText;

    public SupportAnatomy() {
    }

    @Ignore
    public SupportAnatomy(@NonNull long anatomy_id, String anatomy_title, String anatomy_subtitle, String anatomy_text) {
        this.anatomyId = anatomy_id;
        this.anatomyTitle = anatomy_title;
        this.anatomySubtitle = anatomy_subtitle;
        this.anatomyText = anatomy_text;
    }
}

File: app/src/main/java/ru/alexsuvorov/paistewiki/model/SupportAnatomy.kt
package ru.alexsuvorov.paistewiki.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Ignore
import androidx.room.PrimaryKey

@Entity(tableName = "support_anatomy")
class SupportAnatomy {
    @JvmField
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "anatomy_id")
    var anatomyId: Long = 1

    @JvmField
    @ColumnInfo(name = "anatomy_title")
    var anatomyTitle: String? = null

    @JvmField
    @ColumnInfo(name = "anatomy_subtitle")
    var anatomySubtitle: String? = null

    @JvmField
    @ColumnInfo(name = "anatomy_text")
    var anatomyText: String? = null

    constructor()

    @Ignore
    constructor(anatomy_id: Long, anatomy_title: String?, anatomy_subtitle: String?, anatomy_text: String?) {
        this.anatomyId = anatomy_id
        this.anatomyTitle = anatomy_title
        this.anatomySubtitle = anatomy_subtitle
        this.anatomyText = anatomy_text
    }
}